rsts/deployment/cluster_config/performance.rst

```diff
@@ -34,30 +67,52 @@ This is usually because the number of threads in FlytePropeller are not enough t
      - Description
    * - workers
      - propeller
-     - larger the number, more the performance (not too high)
+     - Larger the number, implies more workflows can be evaluated in parallel. But it should depend on number of CPU cores assigned to flytepropeller and evaluated against the cost of context swtiching. A number usually < 500 - 800 with 4-8 cpu cores works fine.
      - Number of `logical threads` workers, that can work concurrently. Also implies number of workflows that can be executed in parallel. Since Flytepropeller uses go-routines, it can run way  more than number of physical cores.
    * - workflow-reeval-duration
      - propeller
-     - lower the number - lower latency, lower throughput
+     - lower the number - lower latency, lower throughput (low throughput is because the same workflow will be evaluated more times)
      - frequency at which, given no external signal, a workflow should be re-evaluated by Flyte propellers reval loop
    * - downstream-eval-duration
      - propeller
-     - lower the number - lower latency and throughput
+     - lower the number - lower latency and lower throughput (low throughput is because the same workflow will be evaluated more times)
      - This indicates how often are external events like pods completion etc recorded.
    * - max-streak-length
      - propeller
-     - higher the number lower the latency for end to end workflow, epsecially for cached workflows
```

Comment:
the maximum number of node transitions performed during a workflow evaluation round